test/compilable/issue15478.d

```diff
@@ -23,3 +23,15 @@ void test2()
     static assert (is(typeof(i2) == int[42]));
 }
 
+// https://issues.dlang.org/show_bug.cgi?id=21870
+struct S
+{
+    @property static size_t count()
```

Comment:
oh this works because `static` is added, fix is not working on the original test case.